src/languages/es.js

```diff
@@ -694,7 +694,8 @@ export default {
     },
     activateStep: {
         headerTitle: 'Habilitar pagos',
-        activated: 'Su billetera Expensify está lista para usar.',
+        activatedTitle: 'Billetera Expensify lista!',
+        activatedMessage: 'Enhorabuena, tu billetera Expensify está lista para usar.',
```

Comment:
Getting confirmation for these [here](https://expensify.slack.com/archives/C21FRDWCV/p1660882913085719).